src/pdc_osti/poster.py

```diff
@@ -173,6 +173,13 @@ def no_empty_cells(series):
     def _fake_post(self, records):
         """A fake JSON response that mirrors OSTI's"""
         self.log.info("[bold yellow]Fake posting")
+        try:
+            ostiapi.datatoxml(records)  # Check that JSON can be parsed into XML
+        except AttributeError:
+            raise AttributeError("Failure to load data into XML!")
+        else:
+            self.log.info("[bold green]Data loaded into XML!")
```

Comment:
```suggestion
        ostiapi.datatoxml(records)  # Check that JSON can be parsed into XML
        self.log.info("[bold green]Data loaded into XML!")
```

Could this be simpler? Either there's an Error, or it info logs.

Reply:
Certainly it can be simplified; however, one of the reasons why I decided to do a `try`/`except` was because I know this will failed in certain circumstances and the traceback is not really helpful (it required a bit of digging to determine which dependency was the problem and identify related GitHub issues). Also, it's common practice to use try/except for flow control.

Perhaps the error message should be more flushed out to communicate why it is failing. For example:
```
"""
Failure to load data into XML!
Check your dicttoxml version (requires > 1.7.4)
"""
```